### PR TITLE
Set session postcode correctly on changing address

### PR DESCRIPTION
**Pull request by whpearson**
Also validate the address is correct before updating session

### CODE REVIEW

File: vulnerable_people_form/form_pages/address_lookup.py [hunk 1]
```diff
@@ -75,15 +75,19 @@ def _create_test_address(postcode):
...
     session["error_items"] = {}
 
     if not validate_address_lookup():
         return redirect("/address-lookup")
 
+    session["form_answers"] = {
+        **session.setdefault("form_answers", {}),
+        "support_address": {**json.loads(request_form()["address"])},
+    }
+    postcode = {**json.loads(request_form()["address"])}.get("postcode", None)
```
Comment (VivianAllen) [inline vulnerable_people_form/form_pages/address_lookup.py:88]: seeing as request_form()["address"] gets parsed into a dict on lines 86 and 88 and 91, is it worth having it done once and stored in a variable?

Comment (sanisc) [reply vulnerable_people_form/form_pages/address_lookup.py:88]: Agree with @VivianAllen

Comment (whpearson) [reply vulnerable_people_form/form_pages/address_lookup.py:88]: Sure

**Review by VivianAllen — APPROVED** [review]
lgtm ✅

**Pull request merged**